bin/angstrom/src/components.rs

```diff
@@ -217,18 +218,22 @@ pub async fn initialize_strom_components<Node: FullNodeComponents, AddOns: NodeA
             .await
             .expect("watch for uniswap pool changes");
     }));
-
-    let price_generator =
-        TokenPriceGenerator::new(provider.clone(), block_id, uniswap_pools.clone())
-            .await
-            .expect("failed to start token price generator");
+    let blocks_to_average_price = 5;
+    let price_generator = TokenPriceGenerator::new(
+        provider.clone(),
+        block_id,
+        uniswap_pools.clone(),
+        Some(blocks_to_average_price)
+    )
+    .await
+    .expect("failed to start token price generator");
```

Comment:
Why did you move this away from the constant. It is a constant in the spec and should always be 5. this implies its configurable which it shouldn't be.